gridstatus/tests/test_ercot.py

```diff
@@ -1084,3 +1142,16 @@ def _check_ercot_spp(df, market, location_type):
         location_types = df["Location Type"].unique()
         assert len(location_types) == 1
         assert location_types[0] == location_type
+
+    def _check_dam_system_lambda(self, df):
+        cols = [
```

Comment:
let's make the order of columns to be consistent with where we put market in other scrapers 

```
cols = [
            "Interval Start",
            "Interval End",
            "Publish Time",
            "Market",
            "System Lambda",
        ]
```

Reply:
Updated.